src/tokenized-line.coffee

```diff
@@ -321,8 +321,11 @@ class TokenizedLine
     return unless maxColumn?
     return unless @text.length > maxColumn
 
-    if /\s/.test(@text[maxColumn])
-       # search forward for the start of a word past the boundary
+    if isCjkCharacter(@text[maxColumn])
+      # always wrap when a CJK character is at the wrap boundary
+      return maxColumn
```

Comment:
I think Chrome render result is fine, break on whitespace and between CJK characters.

![6](https://cloud.githubusercontent.com/assets/49931/11724469/34c1b820-9faf-11e5-8684-01b08083c842.png)

![7](https://cloud.githubusercontent.com/assets/49931/11724530/a6a4ad30-9faf-11e5-8498-75622517cdb9.png)